src/containers/UserMenu/UserMenu.js

```diff
@@ -62,8 +61,50 @@ export class UserMenu extends Component {
                 placeholder="P@$$w0rD"
                 onChange={this.handleChange} />
             </label>
-            <button onClick={(e)=> this.submitForm(e)}>click me</button>
+            <button onClick={(e) => this.submitForm(e)}>click me</button>
+            <button onClick={() => this.setState({ page: '' })}>Back</button>
+          </form >
+      ) 
+    } else if (this.state.page === 'log-in') {
+      return (
+        <form onSubmit={(e) => this.submitForm(e)}>
+        <h2> Log In</h2>
+          <label>
+            Log in (email):
+              <input
+              name="email"
+              value={this.state.email}
+              placeholder="Email"
+              onChange={this.handleChange} />
+          </label>
+          <label>
+            Password:
+              <input
+              name="password"
+              type="password"
+              value={this.state.password}
+              placeholder="P@$$w0rD"
+              onChange={this.handleChange} />
+          </label>
+          <button onClick={(e) => this.submitForm(e)}>click me</button>
+          <button onClick={()=> this.setState({page:''})}>Back</button>
         </form>
+      )
+    } else {
+      return (
+        <form>
+          <button onClick={() => this.setState({ page:'create-account'})}>Create Account</button>
```

Comment:
Let's think about getting rid of the 'create account' button after successful login?